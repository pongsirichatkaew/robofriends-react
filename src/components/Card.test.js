import { shallow, mount, render } from 'enzyme';
import Card from './Card';
import React from 'react';
import toJson from 'enzyme-to-json';

describe('my frist enzyme test', () => {
  it('expect to render Card Component', () => {
    expect(shallow(<Card />)).toMatchSnapshot();
  });
});
