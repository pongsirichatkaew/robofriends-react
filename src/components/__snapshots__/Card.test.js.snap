// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my frist enzyme test expect to render Card Component 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
